constellations/gemini-admin/src/location/locationEdit.sass

```diff
@@ -0,0 +1,41 @@
+@import "../_.constants.sass"
+
+#view-location-edit
+    width: $spacing96x
+    padding: $spacing1x $spacing8x $spacing4x
+    box-sizing: border-box
+
+    & h2
+        margin: $spacing4x 0
+    & h4
+        margin: 0
+
+    & input
+        width: 100%
+        height: $spacing3x
+        margin: $spacing1x 0 $spacing2x
+        font-size: $fontSize0
+
+    & .buttons
+        margin-top: $spacing2x
+        margin-bottom: $spacing8x
+
+    & button
+        padding: $spacing2x $spacing4x
+        border: none
+        border-radius: 6px
+        outline: none
+        font-size: $fontSize0
+        cursor: pointer
+
+        &:hover
```

Comment:
this is the only exception. We need `&:hover` here in order to select the hover state of the parent.